NEWS.md

```diff
@@ -1,3 +1,7 @@
+# Changes in version 2024.10.10 (PR#145)
```

Comment:
please change to PR#154 (145 is the issue)

Reply:
Yes, I changed to PR#154 on my last commit.